src/Symfony/Component/HttpKernel/Tests/Controller/ControllerResolverTest.php

```diff
@@ -196,6 +197,27 @@ public function testGetArguments()
         $this->assertEquals(array($request), $resolver->getArguments($request, $controller), '->getArguments() injects the request');
     }
 
+    /**
+     * @requires PHP 5.6
+     */
+    public function testGetVariadicArguments()
+    {
+        // @see https://github.com/facebook/hhvm/issues/4858
+        if (defined('HHVM_VERSION')) {
+            $this->markTestSkipped('HHVM does not support typed variadics arguments yet. See: https://github.com/facebook/hhvm/issues/4858.');
```

Comment:
actually, we don't even need to have a test with typed variadics: the code you are testing does not care about type hints at all